.gitignore

```diff
@@ -69,6 +69,7 @@ db.sqlite3
 virtualenv
 .vagrant
 docker-compose.yml
+docker-compose.env
```

Comment:
if `docker-compose.env` is being removed from git, should it not be replaced with a `docker-compose.env.example`?